package.json

```diff
@@ -19,6 +19,9 @@
     "number": 8467,
     "sha": "6cb7fec4e154faa0a4a3fee4b33dfef91b9870d9"
   },
+  "config": {
```

Comment:
Needed so puppeteer doesn't auto-download it's own copy of chromium (we take care of that)